Tasks/DownloadPackageV0/connections.ts

```diff
@@ -5,39 +5,22 @@ import { IRequestOptions } from "azure-devops-node-api/interfaces/common/VsoBase
 import * as tl from 'vsts-task-lib/task';
 import * as locationUtility from "packaging-common/locationUtilities";
 
-// TODO Remove this once this bug is resolved: https://github.com/Microsoft/typed-rest-client/issues/126
-export class BearerHandlerForPresignedUrls extends BearerCredentialHandler {
-    prepareRequest(options) {
-        // If we have a presigned blobstore url, don't add auth header
-        if (this.isPreSignedUrl(options)) {
-            delete options.headers["Authorization"];
-            delete options.headers["X-TFS-FedAuthRedirect"];
-        } else {
-            options.headers["Authorization"] = "Bearer " + this.token;
-            options.headers["X-TFS-FedAuthRedirect"] = "Suppress";
-        }
-    }
-
-    isPreSignedUrl(options: any): boolean {
-        return (
-            options.host &&
-            options.host.endsWith("blob.core.windows.net") &&
-            options.path &&
-            options.path.includes("&sig=")
```

Comment:
why are we removing this check?

Reply:
I used the updated library for this task that uses Regex to match the urls. The regex using just the hostname works as well, and its too complicated with additional options.